rebar.config

```diff
@@ -24,13 +24,15 @@
 
 {profiles, [
   {test, [
-    {deps, [ {katana_test, "1.0.1"}
-           , {mixer,       "1.1.1", {pkg, inaka_mixer}}
+    {deps, [ {mixer,       "1.1.1", {pkg, inaka_mixer}}
            , {shotgun,     "0.5.0"}
+           , {xref_runner, "1.2.0"}
```

Comment:
Why do we need `xref_runner`?

Reply:
Oh, `xref_SUITE`… That one can also be killed.

Reply:
Ha, ha, ha, slow laugh, slow clap.

Reply:
Removed.